docs/howtouse.html

```diff
@@ -957,7 +973,227 @@ <h3><a id="sec53">5.3 検索を実行する</a></h3>
 　それからまもなくのこと、王さまがこの村にやってきました。けれども、それが王さまだとは
 </pre>
 
-  <h2><a id="sec6">6. データベースを削除する</a></h2>
+  <h2><a id="sec6">6. Pythonでサンプルを実行する</a></h2>
+  <p>
+    Pythonクライアントライブラリを用いることで、Pythonプログラムからデータベースを操作できます。
+    ライブラリの詳細については<a href="../doquedb-pythondoc/pythondoc/index.html">Python Driver</a>を参照してください。
+  </p>
+  <p>
+    本章のサンプルを実行するためには、以下の条件を満たす必要があります。
+  </p>
+  <ul>
+    <li>Python 3がインストールされていること。</li>
+    <li>py-doquedbがインストールされていること。<br/>
+        手順についてはpy-doquedbの<a href="https://github.com/DoqueDB/doquedb/tree/master/sydney/Python/README.md">README.md</a>を参照してください。「開発環境構築とテスト」の項は実施不要です。</li>
```

Comment:
README link reffers to english document. Since How to Use is written in japanese, link might be better to lead to README_ja.md (japanese documentation)

Reply:
changed to refer Japanese version